hack/lib/sync-api.sh

```diff
@@ -26,48 +26,41 @@ echo "git clone"
 cd ..
 git config --global user.email "openyurt-bot@openyurt.io"
 git config --global user.name "openyurt-bot"
-git clone --single-branch --depth 1 git@github.com:openyurtio/yurt-app-manager-api.git yurt-app-manager-api
+git clone --single-branch --depth 1 git@github.com:openyurtio/api.git yurt-api
 
-if [ -d "yurt-app-manager-api/pkg/yurtappmanager/apis" ]
+if [ -d "yurt-api/apps" ]
 then
-    echo "yurt-app-manager-api apis exists, remove it"
-    rm -r yurt-app-manager-api/pkg/yurtappmanager/apis/*
+    echo "api apps exists, remove it"
+    rm -r yurt-api/apps/*
 else
-    mkdir -p yurt-app-manager-api/pkg/yurtappmanager/apis
+    mkdir -p yurt-api/apps
 fi
 
-if [ -d "yurt-app-manager-api/pkg/yurtappmanager/client" ]
-then
-    echo "yurt-app-manager-api client exists, remove it"
-    rm -r yurt-app-manager-api/pkg/yurtappmanager/client/*
-else
-    mkdir -p yurt-app-manager-api/pkg/yurtappmanager/client
-fi
-
-echo "update yurt-app-manager-api api/"
-cp -R yurt-app-manager/pkg/yurtappmanager/apis/* yurt-app-manager-api/pkg/yurtappmanager/apis/
+echo "update apps"
+cp -R yurt-app-manager/pkg/yurtappmanager/apis/apps/* yurt-api/apps/
 # remove controller depends functions
-rm -r yurt-app-manager-api/pkg/yurtappmanager/apis/apps/v1alpha1/defaults.go
+rm -rf yurt-api/apps/v1alpha1/defaults.go
+
+echo "update addtoschema"
+rm -rf yurt-api/addtoscheme_apps_v1alpha1.go
+cp yurt-app-manager/pkg/yurtappmanager/apis/addtoscheme_apps_v1alpha1.go yurt-api/addtoscheme_apps_v1alpha1.go
```

Comment:
maybe `addtoscheme_apps_v1alpha1.go` should be placed under `yurt-api/apps` dir?

Reply:
The `addtoscheme_apps_v1alpha1 .go` package should be renamed to apps, it's not alligin with origin project.